tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/metrics/AllMetricsTest.java

```diff
@@ -42,11 +43,17 @@
 
     @Deployment
     public static WebArchive deploy() {
+
+        // Scales the following method's annotation values by the TCKConfig baseMultiplier
+        ConfigAnnotationAsset allMetricsBeanConfig = new ConfigAnnotationAsset()
+                .autoscaleMethod(AllMetricsBean.class, "doWork");
+
         WebArchive war = ShrinkWrap.create(WebArchive.class, "ftMetricAll.war")
                 .addClasses(AllMetricsBean.class)
                 .addPackage(Packages.UTILS)
                 .addPackage(Packages.METRIC_UTILS)
-                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml")
+                .addAsManifestResource(allMetricsBeanConfig, "microprofile-config.properties");
```

Comment:
I don't think `addAsManifestResource` works correctly here when the `microprofile-config.properties` is being added to a .war file.

MP Config requires the file to be found on the classpath, whereas I think your code results in the file being located in `META-INF/microprofile-config.properties` which _isn't_ included in the classpath for a .war.

We could place the file elsewhere in the .war, however, could you instead change this deployment method to a a jar and a .war file instead? Most of the tests are packaged this way, it's only some of the metrics tests that aren't. See `TimeoutMetricTest` for an example of a class which does this correctly.

Reply:
I have added a new commit to resolve this issue